lib/ddtrace/writer.rb

```diff
@@ -17,6 +17,7 @@ class Writer
       :worker,
       :events
 
+    # @public_api
     def initialize(options = {})
```

Comment:
I didn't tackle it because the future is `Datadog::Workers::TraceWriter` instead.

I added an internal `DEV:` comment to this class to document this.